src/Package/Impl/ExportDialog/ExportImageDialog.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<ui:PlatformDialogWindow x:Class="Microsoft.VisualStudio.R.Package.ExportDialog.ExportImageDialog"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:pkg="clr-namespace:Microsoft.VisualStudio.R.Package"
+        xmlns:ui="clr-namespace:Microsoft.VisualStudio.R.Package.Wpf"  
+        xmlns:wpf="clr-namespace:Microsoft.R.Wpf;assembly=Microsoft.R.Wpf"
+        xmlns:local="clr-namespace:Microsoft.VisualStudio.R.Package.ExportDialog"
+        xmlns:components="clr-namespace:Microsoft.R.Components;assembly=Microsoft.R.Components"
+        mc:Ignorable="d" MinHeight="150" x:Name="Self"
+        ShowInTaskbar="False" WindowStyle="SingleBorderWindow"
+        Background="{DynamicResource {x:Static wpf:Brushes.BackgroundBrushKey}}"
+        Foreground="{DynamicResource {x:Static wpf:Brushes.UITextKey}}"
+        Title="{x:Static pkg:Resources.ExportImageDialog_Title}" SizeToContent="WidthAndHeight">
+    <Grid Margin="0,0,0,-3">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto"/>
+            <RowDefinition />
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+        <Grid Grid.Row="0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition/>
+                <ColumnDefinition/>
+                <ColumnDefinition Width="auto"/>
+            </Grid.ColumnDefinitions>
```

Comment:
Please make sure all UI is

1. Tested with Dark and Light VS UI themes.
2. Tested with high DPI/Large fonts.
3. Tested with OS High Contrast themes.
4. Fully accessible from keyboard.
5. ESC works as Cancel and Enter as default action.
6. Narrator correctly identifies controls and values to a visually-impaired person.